### PR TITLE
Display

### PR DESCRIPTION
**Pull request by GAUTAMRAJU15**
pls review

### CODE REVIEW

File: src/components/CapturePage/CapturePage.js [hunk 1]
```diff
@@ -63,8 +63,10 @@ class CapturePage extends Component{
 
 
   addPageUrl() {
-    if(this.state.lead.url == '' || !validatePath(this.state.lead.url))
+    if(this.state.lead.url == '')
       return this.setState({error: 'Please enter a valid path'});
+    if(this.state.lead.url[0]!=='/')
```
Comment (ShankyRana) [inline src/components/CapturePage/CapturePage.js]: is this a array or string?

File: src/components/CampaignSettings/CampaignSettings.scss [hunk 2]
```diff
@@ -57,7 +57,7 @@
   }
 }
 
-.btn.btn-custom.goliveRight{
+.btn.btn-outline-primary.goliveRight{
```
Comment (ShankyRana) [inline src/components/CampaignSettings/CampaignSettings.scss]: follow scss standards

File: src/components/Dashboard/Dashboard.js [hunk 3]
```diff
@@ -143,10 +141,11 @@ class Dashboard extends Component {
   }
 
   render() {
-    const { campaignInfo, profile } = this.props;
-
+    const { campaignInfo,profile } = this.props;
+    let datafetch = this.getdays();
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.js]: unnecessary declaration.

File: src/components/Dashboard/Dashboard.js [hunk 4]
```diff
@@ -309,7 +300,7 @@ class Dashboard extends Component {
                           responsive={false}
                           size={100}
                           lineWidth={24}
-                          progress={userSignUps} //to edit
+                          progress={campaignInfo ? campaignInfo.uniqueUsers.length : []} //to edit
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.js]: revert to previous one -> userSignups

File: src/components/Dashboard/Dashboard.js [hunk 5]
```diff
@@ -324,7 +315,7 @@ class Dashboard extends Component {
                       </div>
                       <div className="widget-chart-two-content dash">
                         <span className="text-muted mb-0 mt-2">Total Signups</span>
-                        <h3 className="">{userSignUps}</h3>
+                        <h3 className="">{campaignInfo ? campaignInfo.uniqueUsers.length : []}</h3>
```
Comment (ShankyRana) [inline src/components/Dashboard/Dashboard.js]: revert to previous one -> userSignups

File: src/components/Profile/Profile.scss [hunk 6]
```diff
@@ -15,19 +15,27 @@ button {
   margin: 0 2% !important;
 }
 
-.btn.btn-block.btn-info.waves-light.waves-effect.upgrade1 {
+.btn.btn-block.btn-primary.waves-light.waves-effect.upgrade1 {
```
Comment (ShankyRana) [inline src/components/Profile/Profile.scss]: use scss styling

File: src/components/Profile/Profile.scss [hunk 7]
```diff
@@ -15,19 +15,27 @@ button {
...
+  border-radius: 5px;
 }
 
 .profile-user-box.card-box.bgcolor {
       background-color: rgba(9, 128, 255, 0.152);
     }
 
 
-.btn.btn-block.btn-info.waves-light.waves-effect.billing1 {
+.btn.btn-block.btn-primary.waves-light.waves-effect.billing1 {
```
Comment (ShankyRana) [inline src/components/Profile/Profile.scss]: use scss

File: src/components/LoginFlow/Common.scss [hunk 8]
```diff
@@ -0,0 +1,15 @@
+
```
Comment (ShankyRana) [inline src/components/LoginFlow/Common.scss]: please contain css within a container and give a meaning full name.

File: src/components/CapturePage/CapturePage.js (hunk 1 above)
Comment (Devilla) [reply src/components/CapturePage/CapturePage.js]: It's a string as the string is an array of characters.

File: src/components/Dashboard/Dashboard.js (hunk 5 above)
Comment (GAUTAMRAJU15) [reply src/components/Dashboard/Dashboard.js]: done

File: src/components/Dashboard/Dashboard.js (hunk 4 above)
Comment (GAUTAMRAJU15) [reply src/components/Dashboard/Dashboard.js]: done

File: src/components/Dashboard/Dashboard.js (hunk 3 above)
Comment (GAUTAMRAJU15) [reply src/components/Dashboard/Dashboard.js]: done removed that

File: src/components/Dashboard/Dashboard.js (hunk 5 above)
Comment (GAUTAMRAJU15) [reply src/components/Dashboard/Dashboard.js]: but this gives error

**Pull request closed without merging**